### DISCUSSION TITLE
What is wrong with credentials: 'include'

### DISCUSSION THREAD

**Discussion by decie** [dotnet/aspnetcore#29982] [General]
I use fetch to rsend a post, the config as below:
  `<code>
fetch("url", 
      { 
         mode: 'cors', 
         credentials: 'include', 
          method: 'POST', 
      })
</code>
Without <code> credentials: 'include'</code>, I can send the request successfully, but with <code> credentials: 'include'</code>, I can not  send it. I follow some blog and other solution, but they can not solve this, I set the response header in the bakend:
'
<pre>
    Response.Headers["Access-Control-Allow-Credentials"]="true";
            Response.Headers["Access-Control-Allow-Origin"]="http://[]:8080";
</pre>

**Comment by mrlife** [comment]
I think you will need to create an issue for this or post somewhere like Stack Overflow.

See: https://github.com/dotnet/aspnetcore/issues/29935
